system/interceptors/SES.cfc

```diff
@@ -124,6 +124,22 @@ component extends="coldbox.system.Interceptor" accessors="true"{
 			.setSESBaseURL( variables.baseURL );
 	}
 
+	// CF-11 include .cfm template can't access the methods which are only declare as property... (hack) have to create setter/getter methods
```

Comment:
@sanaullah the entire pull request looks good, except this portion of the setters.  Do you have a reproducible test case for this?